includes/views/advanced-status.php

```diff
@@ -91,7 +91,23 @@
 		'result'   => function_exists( 'base64_decode' ),
 		'fallback' => __( 'base64_decode disabled', 'woocommerce-pdf-invoices-packing-slips' ),
 	),
-) );
+);
+
+if ( $imagick ) {
+	$gmagick_imagick_position = array_search( 'GMagick or IMagick', array_keys( $server_configs ) ) + 1;
+	$image_magick_config      = array(
+		'ImageMagick' => array(
+			'required' => __( 'Required for IMagick', 'woocommerce-pdf-invoices-packing-slips' ),
+			'value'    => ( $imagick && class_exists( '\\Imagick' ) ) ? esc_attr( \Imagick::getVersion()['versionString'] ) : null,
+			'result'   => $imagick,
+			'fallback' => __( 'ImageMagick library, integrated via the IMagick PHP extension for advanced image processing capabilities', 'woocommerce-pdf-invoices-packing-slips' ),
+		),
+	);
```

Comment:
I think you forgot to remove this part from the `$server_configs` array. Now the ImagaMagick line will always show up, also if `$imagick` returns `false`.

Reply:
Sorry, fixed.